packages/lp-glpk-js/lp-glpk-js.0.4.0/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+synopsis: "LP and MIP modeling in OCaml (glpk.js interface)"
+description: """
+This library helps the modeling of Linear Programming (LP) and Mixed Integer Programming (MIP) in OCaml.
+This package is an optional solver-interface to GLPK via glpk.js.
+"""
+maintainer: "Kosuke Tahara <ksk.tahara@gmail.com>"
+authors: [ "Kosuke Tahara <ksk.tahara@gmail.com>" ]
+license: "MIT"
+homepage: "https://github.com/ktahar/ocaml-lp"
+bug-reports: "https://github.com/ktahar/ocaml-lp/issues"
+dev-repo: "git+https://github.com/ktahar/ocaml-lp.git"
+depends: [
+    "ocaml" {>= "4.08.0"}
+    "dune"  {>= "2.2.0"}
+    "lp" {= version}
+    "js_of_ocaml"
+    "js_of_ocaml-ppx"
+    "alcotest" {with-test}
+    "conf-npm" {with-test}
+]
+build: [
+    [ "dune" "build" "-p" name "-j" jobs ]
+    [ "dune" "runtest" "-p" name "-j" jobs ] {with-test}
```

Comment:
```suggestion
#    [ "dune" "runtest" "-p" name "-j" jobs ] {with-test}
```
I am disabling the tests, since they require network access (which is disabled in the sandbox)